lib/pages/people/view/person_view.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:acs_upb_mobile/pages/people/model/person.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class PersonView extends StatelessWidget {
+  final Person person;
+
+  const PersonView({Key key, this.person}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return IntrinsicHeight(
+      child: Column(
+        children: [
+          Container(
+            width: MediaQuery.of(context).size.width,
+            decoration: new BoxDecoration(
+              color: const Color(0xFF43ADCD),
```

Comment:
Don't hard-code colours. Use theme colours (`Theme.of(context).accentColor`). That way, if you ever decide to change the theme, you'll only have to change it in one place.